department-of-veterans-affairs/vets-website
# Add 5103 language to intro and supporting evidence pages

Are you removing, renaming or moving a folder in this PR?

Did you change site-wide styles, platform utilities or other infrastructure?

## Summary
Updates to 526ez for 5103 notice acknowledge
- Introduction page: add a new alert with 'Notice of evidence needed', remove Disability ratings alert, rename the 'VA review' process list title to 'Review', styling changes as per the mocks
- Supporting evidence orientation page: add Notice of evidence needed alert

## Related issue(s)
https://github.com/orgs/department-of-veterans-affairs/projects/1268/views/2?filterQuery=&sliceBy%5Bvalue%5D=Sprint+11&pane=issue&itemId=66470995&issue=department-of-veterans-affairs%7Cva.gov-team%7C83534

## Testing done
- Click testing locally
- Unit tests updated
- e2e tests updated

**Manual test steps**
Note: these changes will only display on dev or lower environments.
1. View the Introduction page both logged out and logged in. **Verify** the page against the mocks. Good idea to verify the page for BDD flow as well.
2. View the supporting evidence orientation page (must be logged in). **Verify** the page against the mocks.

## Screenshots
**Introduction Page**
|         | Before | After |
| ------- | ------ | ----- |
| Mobile|![IntroductionPageMobileBDDBefore](https://github.com/user-attachments/assets/6be88794-0fce-4ce0-9d1b-c81a5403ebcf)|![IntroductionPageMobileBDD](https://github.com/user-attachments/assets/41ee17d8-4744-44b6-9675-a0bc6aae237d)|
| Desktop |![IntroductionPageDesktopBefore](https://github.com/user-attachments/assets/a5e4e8c4-9b58-4c4d-9e0e-b6633f443bfb)|![IntroductionPageDesktopAfter](https://github.com/user-attachments/assets/101882b4-d6d6-46ac-9ba2-df54c36c4152)|

**Supporting evidence Orientation Page**
|         | Before | After |
| ------- | ------ | ----- |
| Desktop |![OrientationDesktopBefore](https://github.com/user-attachments/assets/85af16bd-e7fb-46b1-820f-ebf307546428)|![OrientationDesktopAfter](https://github.com/user-attachments/assets/241296a1-1009-42f3-a1cc-abe09bfd6a0b)|

## What areas of the site does it impact?

Only the 526 form

## Acceptance criteria

### Quality Assurance & Testing


### Error Handling


### Authentication


## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_
